.travis.yml

```diff
@@ -1,18 +1,11 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
```

Comment:
these versions may be EOL, but they're definitely still being used... i don't think we can remove any 7.x version for now, and keeping 5.6 around for a while is probably nice, too. i think we can drop support for them once we actually introduce new features that are not backwards-compatible (or typehints).

Reply:
Removed these because they crash on travis-ci.
I think it's because I bumped phpunit version, I'll drop it back down and see whether they run.